<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PayDialog</class>
 <widget class="qMRMLWidget" name="PayDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>支付窗口</string>
  </property>
  <widget class="QWidget" name="PayDialog2" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>499</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>200</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>32</y>
      <width>258</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:16px;</string>
    </property>
    <property name="text">
     <string>商品名称：升级VIP功能</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPay">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>386</y>
      <width>260</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>立即支付</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>150</y>
      <width>162</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:32px;</string>
    </property>
    <property name="text">
     <string>终身</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>150</y>
      <width>162</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:32px;</string>
    </property>
    <property name="text">
     <string>12个月</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>468</x>
      <y>150</y>
      <width>162</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:32px;</string>
    </property>
    <property name="text">
     <string>1个月</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>52</x>
      <y>215</y>
      <width>14</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:24px;</string>
    </property>
    <property name="text">
     <string>￥</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>215</y>
      <width>14</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:24px;</string>
    </property>
    <property name="text">
     <string>￥</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>488</x>
      <y>215</y>
      <width>14</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:24px;</string>
    </property>
    <property name="text">
     <string>￥</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>208</y>
      <width>101</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:48px;</string>
    </property>
    <property name="text">
     <string>698</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>292</x>
      <y>210</y>
      <width>101</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:48px;</string>
    </property>
    <property name="text">
     <string>198</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>210</y>
      <width>101</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:48px;</string>
    </property>
    <property name="text">
     <string>19.8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPayLife">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>118</y>
      <width>162</width>
      <height>170</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>162</width>
      <height>170</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>162</width>
      <height>170</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPayYear">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>118</y>
      <width>162</width>
      <height>170</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>162</width>
      <height>170</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>162</width>
      <height>170</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPayMonth">
    <property name="geometry">
     <rect>
      <x>468</x>
      <y>118</y>
      <width>162</width>
      <height>170</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>162</width>
      <height>170</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>162</width>
      <height>170</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>80</y>
      <width>181</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:14px;</string>
    </property>
    <property name="text">
     <string>套餐选择：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_order_no">
    <property name="geometry">
     <rect>
      <x>76</x>
      <y>28</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>312</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>支付方式：</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="alipay">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>312</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="text">
     <string>支付宝</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Icons/支付宝支付.svg</normaloff>../Icons/支付宝支付.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QRadioButton" name="wechat">
    <property name="geometry">
     <rect>
      <x>289</x>
      <y>312</y>
      <width>151</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>微信</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Icons/微信支付.svg</normaloff>../Icons/微信支付.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>label_order_no</zorder>
   <zorder>label_4</zorder>
   <zorder>alipay</zorder>
   <zorder>wechat</zorder>
   <zorder>btnPayLife</zorder>
   <zorder>btnPayYear</zorder>
   <zorder>btnPayMonth</zorder>
   <zorder>label</zorder>
   <zorder>btnPay</zorder>
   <zorder>label_3</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>label_8</zorder>
   <zorder>label_9</zorder>
   <zorder>label_10</zorder>
   <zorder>label_11</zorder>
   <zorder>label_12</zorder>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qmrmlwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
